ImageNet/inference/GammaLab_resnet50_PingAnCloud_Caffe.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "v1.0",
+    "author": "PingAn GammaLab & PingAn Cloud team",
+    "authorEmail": "wuxiabao@hotmail.com ，hncstjg@163.com",
+    "framework": "Caffe",
```

Comment:
Can you add the version number for Caffe?

Reply:
OK，we have added the version，and we have also updated a new result of inference latency

Reply:
Thanks!